jre_emul/android/platform/libcore/harmony-tests/src/test/java/org/apache/harmony/tests/java/util/zip/DeflaterOutputStreamTest.java

```diff
@@ -26,10 +26,15 @@
 import java.util.zip.Deflater;
 import java.util.zip.DeflaterOutputStream;
 import java.util.zip.InflaterInputStream;
+//import libcore.junit.junit3.TestCaseWithRules;
+//import libcore.junit.util.ResourceLeakageDetector;
+//import libcore.junit.util.ResourceLeakageDetector.DisableResourceLeakageDetection;
+import org.junit.Rule;
+import org.junit.rules.TestRule;
 
-import junit.framework.TestCase;
-
-public class DeflaterOutputStreamTest extends TestCase {
+public class DeflaterOutputStreamTest extends junit.framework.TestCase /* TestCaseWithRules */ {
```

Comment:
Follow the convention for commenting out lines.

Reply:
added J2ObjC comments